org.lflang/src/org/lflang/generator/PythonGenerator.xtend

```diff
@@ -1053,13 +1053,21 @@ class PythonGenerator extends CGenerator {
         targetConfig.noCompile = true;
         targetConfig.useCmake = false; // Force disable the CMake because 
                                        // it interferes with the Python target functionality
+        var dockerOptions = targetConfig.dockerOptions;
+        targetConfig.dockerOptions = null;
```

Comment:
Perhaps a comment here can help.

If you are overriding `writeDockerFile`, the CGenerator will call your overridden function.